### PR TITLE
Fixes walls becoming nondense due to adminteleport.

### LINKED ISSUE #7594: You can wallk through a wall in the HoP's room
ss 2015-02-07 at 08 22 14

On the east wall, towards the grav gen

### PR DESCRIPTION
**Pull request by theOperand**
Fixed #7594 
#7714 without the entire file being changed.

### CODE REVIEW

File: code/modules/admin/verbs/adminjump.dm [hunk 1]
```diff
@@ -152,6 +152,7 @@
 	var/startdensity = mover.density
 	var/startflags = mover.pass_flags
 	var/startincorporeal = 0
+	var/startdestinationdensity = newloc.density
 	if(istype(mover, /mob/living))
 		var/mob/living/L = mover
 		startincorporeal = L.incorporeal_move
```
Comment (Menshin) [inline code/modules/admin/verbs/adminjump.dm]: Remove everything related to _incorporeal_move_, it's only checked in Client/Move(), hence useless here.

**Comment by Cheridan** [comment]
Weird, why did it need to edit density in the first place?

File: code/modules/admin/verbs/adminjump.dm [hunk 2]
```diff
@@ -152,16 +152,12 @@
 	var/startdensity = mover.density
 	var/startflags = mover.pass_flags
 	var/startincorporeal = 0
```
Comment (phil235) [inline code/modules/admin/verbs/adminjump.dm]: this variable isn't used anymore.

**Comment by phil235** [comment]
Looks good except for that unused var.

**Comment by Miauw62** [comment]
there

this solution actually sucks but ill fix adminjump being broken in another pr

**Pull request merged**